### PR TITLE
Add missing partitioned table setting 'blocks.read_only_allow_delete'

### PR DESCRIPTION
**Pull request by mxm**

### CODE REVIEW

**Comment by mxm** [comment]
Thanks for the review. @chaudum Do you think there are more table settings that need to be added to the partitioned table settings? Asking because this one was missing.

**Comment by chaudum** [comment]
@mxm all regular table settings should also be available for partitioned tables.

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -109,6 +109,9 @@ Changes
 Fixes
 =====
 
+- Made table setting 'blocks.read_only_allow_delete' configurable for
```
[the remark this answers is not in the document]
Comment (chaudum) [reply blackbox/docs/appendices/release-notes/unreleased.rst]: with double backtick I meant ``
so this is formatted as inline code when converted to html

**Comment by mxm** [comment]
@chaudum That doesn't seem to be the case if you look at `TableParameterInfo` vs `PartitionedTableParameterInfo`. We should fix that next.

File: blackbox/docs/appendices/release-notes/unreleased.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (mxm) [reply blackbox/docs/appendices/release-notes/unreleased.rst]: got it

**Comment by mxm** [comment]
Also backported to `3.0` with `7cd35404745da6a6efa3be15e8d27c2265e732db`.

**Pull request merged**